Remotion/Web/Test/Web.Test.csproj

```diff
@@ -1,222 +1,31 @@
-    </Reference>
-    <Reference Include="System.Web.Services" />
-    <Reference Include="System.Xml">
-      <Name>System.XML</Name>
-    </Reference>
-    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj">
-      <Name>Core.Core</Name>
-      <Project>{2AA369EC-A4E7-4D75-9101-EFA90E902625}</Project>
-      <Package>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</Package>
-    </ProjectReference>
-    <ProjectReference Include="..\..\Data\Core\Data.Core.csproj">
-      <Project>{70273934-12F1-49AF-975A-303BAEFE1CAE}</Project>
-      <Name>Data.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\Development\Web\Development.Web.csproj">
-      <Project>{44924ed6-8c12-416d-972f-f1f10417e7ef}</Project>
-      <Name>Development.Web</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\Globalization\Core\Globalization.Core.csproj">
-      <Project>{AC6AB6EB-F6AC-45A4-88A7-877D11A65CCB}</Project>
-      <Name>Globalization.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\Security\Core\Security.Core.csproj">
-      <Project>{FBE0CBDD-82FF-4173-8536-47D3B5BC2E98}</Project>
-      <Name>Security.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Core\Web.Core.csproj">
-      <Name>Web.Core</Name>
-      <Project>{DF8BA786-9679-4879-8757-16AE2709D83A}</Project>
-      <Package>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</Package>
-    </ProjectReference>
-    <ProjectReference Include="..\Test.Shared\Web.Test.Shared.csproj">
-      <Project>{c3639187-f24d-4f9e-9c02-b1bf911a035a}</Project>
-      <Name>Web.Test.Shared</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Global.asax.cs">
-      <DependentUpon>Global.asax</DependentUpon>
-      <SubType>Code</SubType>
-    </Compile>
-    <Content Include="Global.asax">
-      <SubType>Component</SubType>
-    </Content>
-    <Content Include="index.html" />
-    <Content Include="UrlMapping.xml" />
-    <Content Include="Web.config">
-      <SubType>Designer</SubType>
-    </Content>
+    <Content Remove="App.config" />
+    <Content Include="App.config" Link="Web.config" />
   </ItemGroup>
   <ItemGroup>
-    <Service Include="{B4F97281-0DBD-4835-9ED8-7DFB966E87FF}" />
+    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
+    <ProjectReference Include="..\..\Data\Core\Data.Core.csproj" />
+    <ProjectReference Include="..\..\Development\Web\Development.Web.csproj" />
+    <ProjectReference Include="..\..\Globalization\Core\Globalization.Core.csproj" />
+    <ProjectReference Include="..\..\Security\Core\Security.Core.csproj" />
+    <ProjectReference Include="..\Core\Web.Core.csproj" />
+    <ProjectReference Include="..\Test.Shared\Web.Test.Shared.csproj" />
```

Comment:
I'm not sure which depdencies we actually need to declare. please try to minimize?